### PR TITLE
Both test was doing the same but saying something different.

### PR DESCRIPTION
**Pull request by matiaslev**
We have two tests doing the same right now. The goal of this PR is fixed it.

With this fix, the test will work with the log in process as is specified on the name.

### CODE REVIEW

**Review by Serchinastico — APPROVED** [review]
I'm going to wait for @pedrovgs's approval but LGTM

**Comment by pedrovgs** [comment]
You are absolutely right!! Thanks @matiaslev 👏

**Pull request merged**
